### DISCUSSION TITLE
State of Templates (PrimeVue V4 / Nuxt / Tailwind)

### DISCUSSION THREAD

**Discussion by janderegg** [primefaces/community#2844] [PrimeVue Templates] [answered]
I'm looking for a Prime-Template using PrimeVue V4 and also Nuxt, not Vite (great would be Nuxt4-ready with the corresponding folder structure)
I think, this is not available yet, right? Diamond is using PrimeV4 but only Vite. Apollo has Nuxt3, but I think is still on PrimeV3?
Sakai also PrimeV4, but no Nuxt.
I thinks, Tailwind are all templates now, right? 
Great would be: Tailwind/Nuxt/PrimeVueV4. Any plans, if and when this will be available? I think there are quite a few Nuxt-Developers out there...
It would be helpful on the detail pages of the Templates, which versions of which libraries they are build on.

**Comment by cagataycivici** [comment, marked as answer]
All templates are Vue/Vite based not Nuxt but we'll put up a tutorial based on Sakai to demonstrate how easily they can be converted to Nuxt. No plans to provide Nuxt versions out of the box as it gets harder to maintain multiple versions.

**Reply by wisnuind** [reply]
Sorry, can you please help. where I can find the tutorial?

**Reply by thughestamu** [reply]
I am also in this exact situation.  I have an existing project that is built with Primevue and Nuxt and would like to be able to implement one of the primefaces templates.  Any idea as to when we might get this tutorial? or if it already exists where we might find it?

**Reply by kamranata** [reply]
Any updates?

**Reply by spaceark7** [reply]
https://sakai.primevue.org/documentation

Scroll to bottom. Hope it help

**Comment by leosin** [comment]
I'm working on an admin template using primevue+nuxt, just wondering if there's much of a market for this kind of thing.
